src/components/nve-radio-group/nve-radio-group.component.ts

```diff
@@ -59,7 +113,25 @@ export default class NveRadioGroup extends SlRadioGroup {
     return true;
   }
 
-  static styles = [SlRadioGroup.styles, styles];
+  private makeInvalid() {
+    console.log(this.validationMessage);
+    const nveRadios = this.getAllRadios();
+    // toggler 'invalid' attribute på alle radio komponenter for å få riktig style
+    toggleBooleanAttrOnListOfNodes(nveRadios, true, 'data-invalid');
+    if (!this.errorMessageCopy) {
+      this.errorMessageCopy = this.validationMessage;
+    }
+    this.setCustomValidity(`${this.errorMessageCopy}`);
```

Comment:
Må du putte denne i en string? Kan du ikke bruke this.errorMessageCopy direkte?

Reply:
Sant! valgte en rar løsning her